content/en/remote-network-agent/overview.md

```diff
@@ -1,35 +1,45 @@
 ---
-title: Remote Network Agent Installation Overview
+title: Remote Network Agent Overview
 linktitle: Overview
 weight: 1
 categories: ["Remote Network Agent", "Features", "Concepts"]
 tags: ["Remote Network Agent"]
 description: >
-  Learn about the various methods to install an Armory CD-as-a-Service Remote Network Agent in your Kubernetes cluster: kubectl, Helm, Armory CLI, or a UI wizard.
+  a logicless network relay that enables Armory CD-as-a-Service to integrate with privately networked resources such as Jenkins, Prometheus, Kubernetes clusters, etc.
 ---
 
-## Remote Network Agent installation methods
+## Overview
 
-By default, you install a Remote Network Agent (RNA) with full access to your cluster. At a minimum, the RNA needs permissions to create, edit, and delete all `kind` objects that you plan to deploy with CD-as-a-Service, in all namespaces you plan to deploy to. The RNA also requires network access to any monitoring solutions or webhook APIs that you plan to forward through it.
+Armory's Remote Network Agent (RNA) is a simple, lightweight, and reliable agent that resides within your privately networked Kubernetes cluster. It facilitates the orchestration of deployments by providing CD-as-a-Service. RNA enables seamless communication with resources within your private network, empowering CD-as-a-Service to enable use cases such as executing Prometheus queries or initiating Jenkins jobs from within the cluster. RNA supports both `amd64` and `arm64` based architectures, ensuring compatibility across a range of systems.
```

Comment:
i think "privately" could be removed, as we don't make any restrictions about the cluster's location.